src/common/libkvs/test/kvs_txn.c

```diff
@@ -115,25 +115,37 @@ void basic (void)
     rc = flux_kvs_txn_pack (txn, 0, "foo.bar.bleep",  "s", "foo");
     ok (rc == 0,
         "2: flux_kvs_txn_pack(s) works");
+    errno = 0;
+    rc = flux_kvs_txn_pack (txn, 0xFFFF, "foo.bar.blorp",  "s", "foo");
+    ok (rc < 0 && errno == EINVAL,
+        "3: flux_kvs_txn_pack(bad flags) fails with EINVAL");
```

Comment:
ahhh, i thought you were just plainly numbering from 1-9.  So yeah, I'll adjust accordingly.